### PR TITLE
update dataflow diagram

### PR DESCRIPTION
**Pull request by evantahler**
Screen Shot 2021-11-09 at 8 31 54 AM

## Change description

Description here

### Development

- [ ] Application changes have been tested appropriately

### Impact

- [x] Code follows company security practices and guidelines
- [x] Security impact of change has been considered
- [x] Performance impact of change has been considered
- [x] Possible migration needs considered (model migrations, config migrations, etc.)

Please explain any security, performance, migration, or other impacts if relevant:

### Code review

- [x] Pull request has a descriptive title and context useful to a reviewer. Screenshots or screencasts are attached where applicable.
- [x] Relevant tags have been added to the PR (bug, enhancement, internal, etc.)

### CODE REVIEW

**Review by andyjih — COMMENTED** [review]
I wonder if there's a slightly different way to show this... with each Model being a rectangle and within each model has Sources and Destinations. The number of arrows makes things hard to follow for me. I can take a crack at it if you'd like.

**Comment by evantahler** [comment]
@andyjih the good news is that this pulls in the image live from https://docs.google.com/drawings/d/1fXh0WfbBw2iahovnWkhtQ8SBeoX4aVs8liiJRUTZPsM - so any change we make there will be reflected here.

**Comment by evantahler** [comment]
> I wonder if there's a slightly different way to show this... with each Model being a rectangle and within each model has Sources and Destinations. The number of arrows makes things hard to follow for me. I can take a crack at it if you'd like.

Also, since this is a document for developers, I don't want to use the false metaphor that everything lives "under" models, as it's not really the case.  we are going to use that taxonomy in the UI, but I don't think that's appropriate here

**Comment by andyjih** [comment]
Is the target audience external developers or internal developers? If internal, I'm ok with moving on for now.

**Comment by evantahler** [comment]
I guess the audience is our team + those people we've been onboarding to make plugins for/with us.

**Review by andyjih — APPROVED** [review]
